moveit_core/robot_state/test/robot_state_test.cpp

```diff
@@ -560,6 +560,84 @@ TEST_F(OneRobot, testPrintCurrentPositionWithJointLimits)
   state.printStatePositionsWithJointLimits(joint_model_group);
 }
 
+TEST_F(OneRobot, testInterpolation)
+{
+  moveit::core::RobotState state_a(robot_model_);
+
+  // Interpolate with itself
+  state_a.setToDefaultValues();
+  moveit::core::RobotState state_b(state_a);
+  moveit::core::RobotState interpolated_state(state_a);
+  for (size_t i = 0; i <= 10; ++i)
+  {
+    state_a.interpolate(state_b, static_cast<double>(i) / 10., interpolated_state,
+                        robot_model_->getJointModelGroup("base_from_base_to_e"));
+    EXPECT_TRUE(state_a.distance(state_b) < 1e-12);
```

Comment:
Do you mind using EXPECT_NEAR here and defining the epsilon as a const at the top of the file?

Also, do you mind leaving a comment explaining what might have gone wrong if this fails to help anyone who does get this test to fail in the future.  You can do something like this:

```
EXPECT_NEAR(state_a.distance(state_b), 0, 1e-12) << "Reason this might fail";
```

Reply:
Done. I didn't know that was possible with gtest--thanks for showing me